crates/outbound-pg/src/lib.rs

```diff
@@ -238,3 +245,11 @@ fn convert_entry(row: &Row, index: usize) -> Result<DbValue, postgres::Error> {
     };
     Ok(value)
 }
+
+fn spawn(connection: Connection<Socket, NoTlsStream>) {
+    tokio::spawn(async move {
+        if let Err(e) = connection.await {
```

Comment:
is there a reason that we dont want to bubble up this error?

Reply:
We can't bubble it up since `tokio::spawn` is the moral equivalent of spawning a thread -- it will continue to run after our `spawn` function returns.  And the future that `tokio::spawn` accepts returns `()`, so it can't return a `Result`.  Technically we could panic, but that would just leave Tokio to deal with the panic.

Reply:
Could we store a handle to the futures and then await them both?
```
let spawn_handle = spawn_handle(connection);
let client_handle = client
            .execute(statement, params.as_slice());
let nrow, _ = futures::future::try_join_all(vec![spawn_handle, client_handle]).await?;
```

Reply:
To an extent, do we care?  If the connection has failed, the user will get the error from the database request anyway, won't they?

Reply:
true. I'm fine leaving it as. More of a question of whether we want this error to error out the application or not.

Reply:
Yes, based on prior experience with this I can confirm that any connection error that interrupts sending queries or statements or receiving results will throw an error at that level, so no need to process both errors.